src/js/handle-filters-click.js

```diff
@@ -1,12 +1,21 @@
 import axios from 'axios';
 import { constants } from './constants';
 import { createPagination } from './create-pagination';
+import { showSearchForm } from './show-search-form';
+import { setExerciseTitle } from './set-exercise-title';
 
 export async function handleFiltersClick(method) {
   const filterButtons = Array.from(document.querySelectorAll('.btn-filter'));
 
   filterButtons.forEach(button => {
     button.addEventListener('click', async() => {
+
+      const searchForm = document.querySelector('.form-search-exersises');
+      searchForm.reset();
+      searchForm.classList.remove('is-hide');
+      showSearchForm(false);
+      setExerciseTitle('');
```

Comment:
I thing we need to add check if it's exercises page since we don't have search form on category